api/inspector.go

```diff
@@ -125,3 +127,17 @@ func (i *Inspector) PeerStreams() (string, error) {
 	}
 	return string(v), nil
 }
+
+func (i *Inspector) StorageIndices() (string, error) {
+	data, err := i.ls.DebugIndexCounts()
+	if err != nil {
+		return "", err
+	}
+
+	strBytes, err := json.Marshal(data)
```

Comment:
I think you can just return the map, and it will automatically get marshalled.

Reply:
See `DeliveriesPerPeer`.

Reply:
will it get marshalled to a json? or as a string?

Reply:
Depends on the layer above, this is not the layer where we decide if we want to marshal in JSON or another format.

Reply:
It would be nicer if we return map here, but that can be changed later if we want to do testing as soon as possible.